ice.c

```diff
@@ -1431,14 +1447,25 @@ static void janus_ice_webrtc_free(janus_ice_handle *handle) {
 		return;
 	}
 	handle->agent_created = 0;
+#ifdef HAVE_CLOSE_ASYNC
+	nice_agent_attach_recv(handle->agent, handle->stream_id, 1, g_main_loop_get_context(handle->mainloop), NULL, NULL);
+#endif
```

Comment:
We actually stopped doing this in #2101, because of [this issue](https://gitlab.freedesktop.org/libnice/libnice/issues/14) in libnice that is apparently still open. As such, I'd rather not reintroduce it here, unless you know it can't cause the issue presented above.